frappe/api.py

```diff
@@ -82,7 +82,7 @@ def handle():
 
 				if frappe.local.request.method=="PUT":
 					if frappe.local.form_dict.data is None:
-						data = json.loads(frappe.local.request.get_data())
+						data = json.loads(frappe.local.request.get_data().decode('utf-8'))
```

Comment:
i think using [`frappe.safe_decode()`](https://github.com/frappe/frappe/blob/develop/frappe/__init__.py#L1630) would be a better option.